lambdalayer/README.md

```diff
@@ -41,16 +41,19 @@ Environment:
 Conditions:
   UseCodeSigning:
     !Not [!Equals [none, !Ref CodeSigningConfigArn]]
-  IsProd:
-    !Equals [production, !Ref Environment]
 
 Mappings:
-  Constants:
-    DynatraceSecretArn: 
-      Value: !If
-        - IsProd
-        - arn:aws:secretsmanager:eu-west-2:216552277552:secret:DynatraceProductionVariables
-        - arn:aws:secretsmanager:eu-west-2:216552277552:secret:DynatraceNonProductionVariables
+  EnvironmentConfiguration:
```

Comment:
This is probably not the nicest way to do it, but couldn't get the if statement working, my knowledge of cloudformation/sam is very minimal and this fit well with what we already had in our template